File: Map/src/app/shared/DanhMucHieuMay.service.ts
﻿import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucHieuMay } from 'src/app/shared/DanhMucHieuMay.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucHieuMayService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucHieuMay";
    }
}

File: Service/Implement/DanhMucHieuMayService.cs
namespace Service.Implement
{
    public class DanhMucHieuMayService : BaseService<DanhMucHieuMay, IDanhMucHieuMayRepository>
    , IDanhMucHieuMayService
    {
    private readonly IDanhMucHieuMayRepository _DanhMucHieuMayRepository;
    public DanhMucHieuMayService(IDanhMucHieuMayRepository DanhMucHieuMayRepository) : base(DanhMucHieuMayRepository)
    {
    _DanhMucHieuMayRepository = DanhMucHieuMayRepository;
    }
    }
    }

